dashboard/app/models/contact_rollups_pardot_memory.rb

```diff
@@ -31,50 +31,89 @@ def self.add_and_update_pardot_ids(last_id = nil)
     PardotV2.retrieve_new_ids(last_id).each do |mapping|
       pardot_record = find_or_initialize_by(email: mapping[:email])
       pardot_record.pardot_id = mapping[:pardot_id]
-      pardot_record.pardot_id_updated_at = Time.now
+      pardot_record.pardot_id_updated_at = Time.now.utc
       pardot_record.save
     end
   end
 
   def self.create_new_pardot_prospects
+    # Adds contacts to a batch and then sends batch requests to create new Pardot prospects.
```

Comment:
Hard to diff these, there's no change here other than moving the SQL?

Reply:
That's correct. I should have separated changes to syncing new prospects and updating existing prospects.